### PR TITLE
Add namespace from Phinx config to migration

### LINKED ISSUE #86: Namespace for migrations does not work
Phinx [supports custom namespaces](https://phinx.readthedocs.io/en/latest/namespaces.html), e.g.
```
    'paths' => [
        'migrations' => [
            'Db\Migrations' => '%%PHINX_CONFIG_DIR%%/db/migrations',
        ],
    ],
```

With this configuration, phinx-migrations-generator does not work.

`In GenerateCommand.php line 203: No migration paths set in your Phinx configuration file.`

**Versions**
Tested with `"robmorgan/phinx": "^0.11.4"` as well as `"robmorgan/phinx": "^0.12.4"`
Using `"odan/phinx-migrations-generator": "^5.1"`
 
**Cause**
Use of `$migrationsPaths[0]` while the key is a string

### PR DESCRIPTION
**Pull request by GlennM**
Fixes #86

### CODE REVIEW

**Comment by GlennM** [comment]
There's one failing test, though I am not able to fix it. Feel free to adjust where needed.

File: src/Migration/Command/GenerateCommand.php [hunk 1]
```diff
@@ -199,11 +200,13 @@ private function getGeneratorSettings(InputInterface $input, string $environment
         $migrationsPaths = is_array($migrationsPaths) ? $migrationsPaths : (array)$migrationsPaths;
 
         // No paths? That's a problem.
-        if (empty($migrationsPaths[0])) {
+        if (count($migrationsPaths) === 0) {
```
Comment (odan) [inline src/Migration/Command/GenerateCommand.php:203]: Now this line could cause unwanted warnings. Please revert it.

File: src/Migration/Adapter/Generator/PhinxMySqlGenerator.php [hunk 2]
```diff
@@ -83,6 +83,10 @@ public function createMigration($name, $newSchema, $oldSchema): string
 
         $output = [];
         $output[] = '<?php';
+        if (!is_null($this->options['namespace'])) {
```
Comment (odan) [inline src/Migration/Adapter/Generator/PhinxMySqlGenerator.php:86]: This cuases warnings when the index is not set. Please use `if (!empty($this->options['namespace'])) {` instead.

Comment (GlennM) [reply src/Migration/Adapter/Generator/PhinxMySqlGenerator.php]: Done

File: src/Migration/Command/GenerateCommand.php (hunk 1 above)
Comment (GlennM) [reply src/Migration/Command/GenerateCommand.php]: Adjusted, not 100% reverted as index `0` does not exist on the array when using namespaces.

**Pull request merged**